Lib/configparser.py

```diff
@@ -1027,36 +1056,61 @@ def _read(self, fp, fpname):
                             elements_added.add(sectname)
                         # So sections can't start with a continuation line
                         optname = None
-                    # no section header in the file?
+                    # no section header?
                     elif cursect is None:
                         raise MissingSectionHeaderError(fpname, lineno, line)
-                    # an option line?
+                        # an option line?
```

Comment:
This change in indentation seems to be a mistake, since the scope of the comment is meant to be the else clause below it.

```suggestion
                    # an option line?
```